package.json

```diff
@@ -45,7 +46,7 @@
     "ember-maybe-import-regenerator-for-testing": "^1.0.0",
     "ember-moment": "^7.5.0",
     "ember-resolver": "^4.4.0",
-    "ember-source": "https://s3.amazonaws.com/builds.emberjs.com/canary/shas/6f92209039f4f9a2c08c5642a02a4e598f3702cd.tgz",
+    "ember-source": "~3.1.0-beta.1",
```

Comment:
I'm confused by this, shouldn't my PR have this change? 🤔

Reply:
@locks I think yours was still on canary.